client/src/Pages/EditAddress.elm

```diff
@@ -137,8 +145,8 @@ update key msg model authStatus details =
                     ( model, Cmd.none )
 
 
-selectAddress : AddressId -> Form -> List Address.Model -> Form
-selectAddress ((AddressId i) as id) model addresses =
+selectAddress : AddressId -> Form -> Bool -> List Address.Model -> Form
+selectAddress ((AddressId i) as id) model selectedAddressShipping addresses =
```

Comment:
Do we allow user to submit an address that doesn't exist in `postgrid`. I think we should allow, because there is always an address that is somewhere strange.

And with that, we can store a `is_verified` flag instead to show if an address is verified via `postgrid` or not.

Reply:
Yeah, there will be an extra validation step on the backend side, and address will be marked as verified in the DB only as soon as it was actually verified there

Reply:
This PR now is only a frontend draft :sweat_smile:

Reply:
> I think we should allow, because there is always an address that is somewhere strange.

We shouldn't allow using an address that is not verified for delivery as a shipping address. Postgrid targets to verify all addresses valid for USPS

Reply:
@rvem  Can we ask the client about this 🤔, with this, we will rely too much on Postgrid, and if their api is down or they make a mistake somewhere, we might lose customer since they cannot do the checkout.

Reply:
> and if their api is down

Their SLA is "99.9% of the time during each calendar month (the “Monthly Uptime Percentage”)" according to https://www.postgrid.com/legal/

Reply:
>  The service uptime guarantee does not apply to service interruptions caused by: (1) periodic scheduled maintenance or repairs we may undertake from time to time; (2) interruptions caused by you or your activities; (3) outages that do not affect core Service functionality; (4) causes beyond our control or that are not reasonably foreseeable; and (5) outages related to the reliability of certain programming environments

🤔 It doesn't include other things, so I'd say 95% of all the time.

Reply:
> we might lose customer since they cannot do the checkout.

OTOH, we might as well lose a customer if they provide an invalid delivery address. As a result, the payment was done, but we, as a shop, are unable to fulfill our obligations on the delivery. And in fact, this was already the case multiple times, according to Scott

Reply:
> so I'd say 95% of all the time.
I'm looking at https://status.postgrid.com/incidents/, and it doesn't seem too bad.

In general, I agree that it makes sense to have a fallback for PostGrid API unavailability (e.g. do not mark an address as verified in the DB but still proceed with the checkout), but I don't think we should allow checkouts with shipping addresses that PostGrid failed to verify